traffic_ops/traffic_ops_golang/deliveryservice/health_test.go

```diff
@@ -0,0 +1,165 @@
+package deliveryservice
+
+/*
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import (
+	"github.com/apache/trafficcontrol/lib/go-tc"
+
+	"encoding/json"
+	"testing"
+)
+
+const crConfigJson = `
```

Comment:
Instead of making two JSON string constants and then doing nothing with them but unmarshalling into a real struct, why not just create that struct immediately, so you get compile-time type safety?